### PR TITLE
Remove limit on template params length

### PR DESCRIPTION
**Pull request by margalva**
Remove limit on parameters input field. Work-around while a better implementation is designed.

### CODE REVIEW

File: src/ansys/dynamicreporting/core/utils/report_objects.py [hunk 1]
```diff
@@ -1465,7 +1465,6 @@ def get_json_key_limits(cls):
         d = super().get_json_key_limits()
         d["name"] = 255
         d["report_type"] = 50
-        d["params"] = 4096
```
Comment (viseshrp) [inline src/ansys/dynamicreporting/core/utils/report_objects.py:1468]: yeah, hardcoded limits like these should be done away with and ideally the server should tell you if you have exceeded the limit in any case @margalva

**Review by viseshrp — APPROVED** [review]

**Review by randallfrank — COMMENTED** [review]
@margalva   @viseshrp You are not concerned about backward compatibility here?  I was just thinking that the maximum length is server API specific, assuming the server protocol number changed with the maximum length change.

**Comment by margalva** [comment]
@randallfrank As we're opening up the limit so now it's larger than it used to, it shouldn't pose backward compatibility issues, as far as I see it. Old cases will work with the new API. New cases won't work against old APIs, but that's not the suggested direction anyway - we don't even let you start a server from an old version on top of a new database.

**Comment by viseshrp** [comment]
@randallfrank I fail to see the backwards incompatibility here as it is just an extension of the number of allowed chars, as @margalva has mentioned.

**Pull request merged**